### DISCUSSION TITLE
Error opening excel, Records removed: Style from /xl/styles.xml part (Styles)

### DISCUSSION THREAD

**Discussion by Kristianpk** [qax-os/excelize#1976] [Q&A]
After procesing a file.xlsx, when I open the new file (processed with excelize) I'm getting this Error, Do you know why? 
The process that I'm doing is replacing some cell with new values from a csv file. I'm reading the csv file and saving the information in a matrix and then I go through the cells of the file and replace the values. 
I've checked the style.xml file and verifyed with an xml reader and everything it's OK
I'll aprecciate so much your help

**Comment by xuri** [comment]
Thanks for your feedback. Which version of the Excelize library are you using? Could you show us a complete, standalone example program or reproducible demo? If you open an existing workbook, please provide the file attachment without confidential info.

**Comment by Kristianpk** [comment]
Hi, I'm using "go version go1.18.2 linux/amd64", For sure, I'll send the code and the files.
Attached you will find:

  1.
The .go file where we're using the excelize library to ready and create a new xlsx file to procces some tags and replace them for a value.
  2.
The .csv file where we're processing the tags ("[cmp_XX]") and the new values.
  3.
The .txt(csv with ";" delimiter) file, here comes the tags and the values that will be replaced on the .xlsx file

So after processing the xlsx and save the new file when we open it we have this error.

[cid:70918854-d1b1-4a4e-a71f-85d864f1c164]
[cid:cb0a2129-91c4-419f-87b9-9390e339c3c8]
The procces of replace the values it's ok, but we're having this issues.

I'll appreciate so much your help

Best regards.


________________________________
De: xuri ***@***.***>
Enviado: jueves, 22 de agosto de 2024 7:36 p. m.
Para: qax-os/excelize ***@***.***>
Cc: Kristianpk ***@***.***>; Author ***@***.***>
Asunto: Re: [qax-os/excelize] Error opening excel, Records removed: Style from /xl/styles.xml part (Styles) (Discussion #1976)


Thanks for your feedback. Which version of the Excelize library are you using? Could you show us a complete, standalone example program or reproducible demo? If you open an existing workbook, please provide the file attachment without confidential info.

—
Reply to this email directly, view it on GitHub<https://github.com/qax-os/excelize/discussions/1976#discussioncomment-10424801>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AHZUEHNM3M5PC6W33AJFIPLZSZ7XPAVCNFSM6AAAAABM6ODZB2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBSGQ4DAMI>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

cmp_5;156498165
cmp_7;Torres Correa Cristian
cmp_11;Cristian Camilo LLC
cmp_25;555555555
cmp_30;
cmp_PxSala;111222255555

**Reply by xuri** [reply]
Could you show us a complete, standalone example program code with main function?

**Comment by Kristianpk** [comment]
The version I've sent you has main function, I've sent you attached again.
________________________________
De: xuri ***@***.***>
Enviado: lunes, 26 de agosto de 2024 10:33 p. m.
Para: qax-os/excelize ***@***.***>
Cc: Kristianpk ***@***.***>; Author ***@***.***>
Asunto: Re: [qax-os/excelize] Error opening excel, Records removed: Style from /xl/styles.xml part (Styles) (Discussion #1976)


Could you show us a complete, standalone example program code with main function?

**Reply by xuri** [reply]
I didn't saw the attachment. Please confirm the attachment uploaded successfully.

**Comment by Kristianpk** [comment]
Hi, the attachement it's OK

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by xuri** [reply]
I still didn't see the attachment. Could you follow the issue template provide more details of this and the upload an attachment without confidential info?
